owned-manifests/machine-api-controllers.yaml

```diff
@@ -36,18 +36,18 @@ spec:
         key: node.alpha.kubernetes.io/unreachable
         operator: Exists
       containers:
-      - name: controller-manager
+      - name: machineset-controller-manager
         image: {{ .Controllers.Provider }}
         command:
-          - "./manager"
+          - /manager
         args:
           - --logtostderr=true
           - --v=3
         resources:
           requests:
             cpu: 10m
             memory: 20Mi
-      - name: machine-controller
+      - name: machine-controller-manager
```

Comment:
-1. Either `controller` or just `manager`. Yes, we built on top of controller-runtime which defines a concept of a manager. Yet, we run only one controller. It only make sense to either use `machine-controller` or `machine-manager`. What's the motivation behind the renaming? `Better naming` does not explain it much.

Reply:
*-controller-manager is the most accurate, self explanatory and in line with core components naming https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/

Reply:
You refer to only one component name that ends with `-controller-manager` suffix. How does it generalizes to other component names? Plus, name `kube-controller-manager` is very pre-historic name from the middle ages of kubernetes when we used swords and sticks to do all the magic.

Reply:
not sure I follow, this is literally a controller manager that might wrap one or more Controllers

Reply:
e.g https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/cmd/clusterctl/examples/aws/provider-components-base/kustomization.yaml#L10
e.g https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/

Reply:
changed to *-controller discussed offline to be more clear for the majority